worlds/hylics2/__init__.py

```diff
@@ -132,21 +167,14 @@ def generate_basic(self):
                 tv = self.multiworld.random.choice(tvs)
                 self.multiworld.get_location(tv[1]["name"], self.player)\
                     .place_locked_item(self.add_item(gest[1]["name"], gest[1]["classification"], gest[1]))
+                self.pre_fill_items.append(\
+                    self.add_item(gest[1]["name"], gest[1]["classification"], gest[1]))
                 gestures.remove(gest)
                 tvs.remove(tv)
 
-        else: # add gestures to pool like normal
-            for i, data in Items.gesture_item_table.items():
-                pool.append(self.add_item(data["name"], data["classification"], i))
-
-        # add '10 Bones' items if medallion shuffle is enabled
-        if self.multiworld.medallion_shuffle[self.player]:
-            for i, data in Items.medallion_item_table.items():
-                for j in range(data["count"]):
-                    pool.append(self.add_item(data["name"], data["classification"], i))
 
-        # add to world's pool
-        self.multiworld.itempool += pool
+    def get_pre_fill_items(self) -> List["Item"]:
```

Comment:
you don't seem to actually use this for anything? the use of this method is that you would actually build all the items you need to place during pre_fill *in* this method, which it then returns as a list. you can then either cache it and reference it as necessary unless you need it multiple times.